src/main/kotlin/com/sourcegraph/cody/chat/AgentChatSession.kt

```diff
@@ -10,13 +10,14 @@ import com.sourcegraph.cody.agent.*
 import com.sourcegraph.cody.agent.protocol.*
 import com.sourcegraph.cody.chat.ui.ChatPanel
 import com.sourcegraph.cody.commands.CommandId
+import com.sourcegraph.cody.config.CodyAccount.Companion.isEnterpriseAccount
 import com.sourcegraph.cody.config.CodyAuthenticationManager
 import com.sourcegraph.cody.config.RateLimitStateManager
 import com.sourcegraph.cody.history.HistoryService
 import com.sourcegraph.cody.history.state.ChatState
 import com.sourcegraph.cody.history.state.MessageState
 import com.sourcegraph.cody.ui.ChatModel
-import com.sourcegraph.cody.ui.CodyModelComboboxItem
+import com.sourcegraph.cody.ui.LLMModelComboBoxItem
```

Comment:
nit: do we really need to change the name here? If so i would even suggest `LLMComboBoxItem` since LLM stands for large language model.

Reply:
"Cody" part is redundant. good point about "Model"